viewport/index.js

```diff
@@ -1 +1,45 @@
-module.exports = require('o-viewport');
+const oViewport = require('o-viewport');
+const utils = require('../utils');
+
+const GRID_BREAKPOINTS = new Map([
+	['default', [0,489]],
+	['S', [490,739]],
+	['M', [740,979]],
+	['L', [980,1219]],
+	['XL', [1220,Infinity]]
+]);
```

Comment:
I just realised o-grid (which should be bundled) has a `getCurrentLayout` method so we probably don't need to dupe the breakpoints